Date: 2005-08-27
Initial Package Version: 4.6.1-pre4
Upstream Status: Not yet submitted.
Origin: Miklos Vajna <vmiklos@frugalware.org>
Description: Adds the ability to hit enter on .fpm or FrugalBuild files.
diff -Naur mc-4.6.1-pre4.orig/lib/mc.ext.in mc-4.6.1-pre4/lib/mc.ext.in
--- mc-4.6.1-pre4.orig/lib/mc.ext.in	2005-06-07 18:54:00.000000000 +0200
+++ mc-4.6.1-pre4/lib/mc.ext.in	2005-08-27 21:37:26.000000000 +0200
@@ -106,8 +106,8 @@
 
 ### Archives ###
 
-# .tgz, .tpz, .tar.gz, .tar.z, .tar.Z
-regex/\.t([gp]?z|ar\.g?[zZ])$
+# .tgz, .tpz, .tar.gz, .tar.z, .tar.Z, .fpm, .fdb
+regex/\.(t([gp]?z|ar\.g?[zZ])|f(pm|db))$
 	Open=%cd %p#utar
 	View=%view{ascii} gzip -dc %f 2>/dev/null | tar tvvf -
 
@@ -504,6 +504,10 @@
 regex/[Mm]akefile$
 	Open=make -f %f %{Enter parameters}
 
+# FrugalBuild
+regex/FrugalBuild$
+	Open=makepkg -p %f %{Enter parameters}
+
 # Imakefile
 shell/Imakefile
 	Open=xmkmf -a
